tests/image/test_push.py

```diff
@@ -42,30 +43,18 @@ def test_cli_with_progress(docker_client_mock, test_mock):
         {"status": "Pushing", "progressDetail": {"current": 512, "total": 24803}, "id": "abc"},
         {"status": "Layer already exists", "progressDetail": {}, "id": "def"},
         {"status": "Pushing", "progressDetail": {"current": 57344, "total": 26348}, "id": "egh"},
+        {"status": "Pushed", "progressDetail": {}, "id": "egh"},
         {"status": "Pushing", "progressDetail": {"current": 24805, "total": 24803}, "id": "abc"},
         {"status": "Pushed", "progressDetail": {}, "id": "abc"},
         {"status": "Successfully pushed"}
     ]
     runner = CliRunner()
     result = runner.invoke(cli, ["dev", "--version", "test"])
     assert result.exit_code == 0
-    import re
-    assert re.match(
-        'Login to registry succeeded.\n'
-        'Pushing registry/user/project:test to the registry.\n\r'
-        'Layers:   0%|          | 0/1\r          \r'
-        'Layers:   0%|          | 0/1\r          \r'
-        'Layers:   0%|          | 0/2\r          \r'
-        'Layers:   0%|          | 0/3\r          \r'
-        'Layers:   0%|          | 0/3\r          \r'
-        'Layers:   0%|          | 0/3\n\r'
-        'abc:   0%|          | 0.00/24.8K [?B/s]\x1b[A\r'
-        '          \r'
-        'Layers:   0%|          | 0/3\n\n\r'
-        'egh:   0%|          | 0.00/26.3K [?B/s]\x1b[A\x1b[A\r'
-        '          \r'
+    assert re.search(
```

Comment:
Because progress speed (is a part of bar) depends on time and it's always different